plugin-gradle/src/main/java/com/diffplug/gradle/spotless/ScalaExtension.java

```diff
@@ -48,21 +48,29 @@ public ScalaFmtConfig scalafmt(String version) {
 	public class ScalaFmtConfig {
 		final String version;
 		@Nullable
+		String scalaMajorVersion;
+		@Nullable
 		Object configFile;
 
 		ScalaFmtConfig(String version) {
 			this.version = Objects.requireNonNull(version);
-			addStep(createStep());
 		}
 
-		public void configFile(Object configFile) {
+		public ScalaFmtConfig configFile(Object configFile) {
```

Comment:
This had to be changed to from `void` to `ScalaFmtConfig` otherwise you can't chain the `configFile`/`scalaMajorVersion` methods in builder pattern style.